allure-cucumber6-jvm/src/main/java/io/qameta/allure/cucumber6jvm/AllureCucumber6Jvm.java

```diff
@@ -99,6 +100,9 @@ public class AllureCucumber6Jvm implements ConcurrentEventListener {
 
     private static final String TXT_EXTENSION = ".txt";
     private static final String TEXT_PLAIN = "text/plain";
+    private static final char CUCUMBER_RESOURCE_SEPARATOR_CHAR = '/';
+    private static final String CUCUMBER_WORKING_DIR = System.getProperty("user.dir")
```

Comment:
maybe use `new File("")` instead?

Reply:
I found that Cucumber uses `io.cucumber.core.resource.Resources$UriResource`to get the test case uri, which in turn uses `Path` instead of `File`. Because Path#toUri and File#toURI gets some different result, I did it with Paths.get("")